cmd/load-runner/helm.go

```diff
@@ -33,9 +34,13 @@ func (h *helmClient) Install(ctx context.Context, config *parser.LoadTestConfig)
 	}
 
 	for key, value := range config.EnvOverrides {
+		if strings.Contains(value, ",") {
+			value = strings.ReplaceAll(value, ",", "\\,")
+		}
```

Comment:
This seems like a functionality needed in different places. Should we extract a function and add it to the parser?

Reply:
done. this is specific to helm, so i created a dedicated function to handle env vars.